backend/schema/corpora_schema_h5ad_implementation.md

```diff
@@ -0,0 +1,90 @@
+# Corpora Schema AnnData Implementation
```

Comment:
it would be good to have a section in this doc on how a tool can unambiguously determine if a dataset _is_ a Corpora dataset.  Just looking for `uns.version` seems likely to trip over other uses of that generic name.

Reply:
Added a note